### ISSUE TITLE
More options for fallback behaviour

### ISSUE THREAD

**Issue by JasonBarnabe** [gitcoinco/code_fund_ads#927]
As a publisher, I'm fine with the general concept of showing fallback ads for CodeFund's benefit, but looking at my stats I can see some properties on some days showing 60% fallback ads. This is significant traffic that I'm not directly benefiting from, so I would like more options on how it is handled.

What I would like is the option to have CodeFund inject HTML of my choosing as a fallback. I could then use this to insert different ad code. To keep some benefit to CodeFund, perhaps I could set a percentage to do CodeFund's fallback, with the rest doing my provided HTML.

**Comment by coderberry** [comment]
@JasonBarnabe I agree 100%. This is something that our competitors offer which we really love. The workaround right now is to use our callback script ([see here](https://github.com/gitcoinco/code_fund_ads#embed-callbacks)) to render code on your end when there are no paid ads to show.

We will add this to our Q2 agenda.

**Comment by JasonBarnabe** [comment]
@coderberry Using the callback script works for what I want to do and is actually less of a workaround than my initial idea (I was just going to put script in the fallback HTML anyway). So I would not use a fallback HTML feature if it were developed.
